packages/contracts/contracts/tokens/GovernanceWrappedERC20.sol

```diff
@@ -37,11 +37,11 @@ contract GovernanceWrappedERC20 is Initializable, AdaptiveERC165, ERC20VotesUpgr
         return "0.0.1+opengsn.recipient.GovernanceWrappedERC20";
     }
 
-    function initialize(
+    function __GovernanceWrappedERC20_init(
```

Comment:
what's the reasoning behind bringing __GovernanceWrappedERC20_init inside initialize ? just tried to put all the code that is inside __GovernanceWrappedERC20_init in the initialize function directly and it works.

could you explain or have it the same way we had ?

Reply:
I wanted to maintain the possibility to inherit from the token, but I guess that's not needed, and we can save gas this way. I reverted the change.